### ISSUE TITLE
Only copies to ios/Assets, not copying into the .xcworkspace or .xcodeproj

### ISSUE THREAD

**Issue by doyeka** [Malaa-tech/expo-custom-assets#6]
I used this solution because I thought it would just work™ .

However, copying to ios/Assets doesn't add those files to the XCode project (either .xcworkspace or .xcodeproj).

Thus, the 'Copy Bundle Resources' phase of the build process fails.

Can you update the code base to copy to within the XCode project?

![Screenshot 2024-06-24 at 1 04 32 PM](https://github.com/Malaa-tech/expo-custom-assets/assets/23086057/84c5de71-7d20-443d-a3e6-98cdf06c607b)

**Comment by doyeka** [comment]
Yeah, seems like using [`IOSConfig.XcodeUtils.addResourceFileToGroup`](https://github.com/Malaa-tech/expo-custom-assets/blob/4bbff58728259cd546da13cc11ad074fb5059990/src/index.ts#L68) is broken for now, via: https://github.com/expo/config-plugins/issues/239

Will just manually add "Assets" to my target for now.

Subscribed to the other issue, happy to implement the fix once it's known.

**Comment by doyeka** [comment]
Info about why it's broken here: https://github.com/expo/expo/issues/20091

**Comment by jottenlips** [comment]
This would greatly simplify our build process and for anyone using fonts with expo-widgets
